### PR TITLE
Updated Discord Botlist

### PR DESCRIPTION
**Pull request by Iserila**
- Updated **Discord Botlist** to work with the new updated website paths
- Updated metadata: logo, banner, version, _contributers_

### CODE REVIEW

**Comment by Exstare** [comment]
CodeFactor did some 'autofixes', it messed a lot up

File: websites/D/Discord BotList/dist/metadata.json [hunk 1]
```diff
@@ -5,17 +5,28 @@
...
+      "name": "Exstare",
+      "id": "462914535351779328"
+    }
+  ],
   "category": "other",
-  "logo": "https://i.imgur.com/V8TuIHR.jpg",
-  "thumbnail": "https://i.imgur.com/utk2huo.png",
-  "version": "1.0.4",
+  "logo": "https://imgur.com/HwNmdMj.png",
+  "thumbnail": "https://imgur.com/R24meTP.png",
```
Comment (Bas950) [inline websites/D/Discord BotList/dist/metadata.json:16]: use i.imgur links

File: websites/D/Discord BotList/dist/metadata.json [hunk 2]
```diff
@@ -5,17 +5,28 @@
...
     "discord",
     "botlist",
     "list",
     "discord-botlist"
   ],
+  "altnames": [
+    "discord-botlist",
+    "discord-botlist.eu",
+    "dbleu"
+  ],
```
Comment (Bas950) [inline websites/D/Discord BotList/dist/metadata.json:29]:
```suggestion
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working

File: websites/D/Discord BotList/presence.ts [hunk 3]
```diff
@@ -1,121 +1,150 @@
...
   if (botHost == "discord-botlist.eu") {
-    if (botPage == "/") {
-      botData.details = "Browsing";
-    } else if (botPage == "/search") {
+    if (botPage === "/") 
+
+      botData.details = "Browsing home";
+
+     else if (botPage === "/search") {
+
```
Comment (ImRodry) [inline websites/D/Discord BotList/presence.ts:22]: You don't need this many empty lines, you can just put the code on the same line as the if check

File: websites/D/Discord BotList/presence.ts [hunk 4]
```diff
@@ -1,121 +1,150 @@
...
-        myParam: string = urlParams.get("tag");
-      botData.details = "Searching Tag:";
-      botData.state = myParam;
+
+    } else if (botPage === "/login_err/") 
+
+      botData.details = "Login page";
+
+     else if (botPage.includes("/users/")) {
+
```
Comment (ImRodry) [inline websites/D/Discord BotList/presence.ts:40]: Same as above

File: websites/D/Discord BotList/presence.ts [hunk 5]
```diff
@@ -1,121 +1,150 @@
...
-      const username: string = document.querySelector("#Username").textContent;
-      botData.details = "Viewing Profile:";
-      botData.state = username;
+
+    } else if (botPage.includes("/submit")) 
+
+      botData.details = "Submitting a new bot";
+
+     else if (botPage.includes("/vote")) {
+
```
Comment (ImRodry) [inline websites/D/Discord BotList/presence.ts:56]: ditto

File: websites/D/Discord BotList/presence.ts [hunk 6]
```diff
@@ -1,121 +1,150 @@
...
+    } else if (botPage.includes("/edit")) {
+
+      const editBotName: string = document
+        .querySelector("#name").textContent;
+      botData.details = "Editing a bot";
+      botData.state = editBotName;
+
+    } else if (botPage.includes("/all")) {
+
```
Comment (ImRodry) [inline websites/D/Discord BotList/presence.ts:77]: Ditto

File: websites/D/Discord BotList/presence.ts [hunk 7]
```diff
@@ -1,121 +1,150 @@
...
+          label: "View Bot",
+          url: document.URL
+        }
+      ];
+
+    } else if (botPage === "/manage/me") {
+
+      const page: string = document
+        .querySelector("#content-manage > strong").textContent;
+
```
Comment (ImRodry) [inline websites/D/Discord BotList/presence.ts:102]: ditto

File: websites/D/Discord BotList/presence.ts [hunk 8]
```diff
@@ -1,121 +1,150 @@
...
+       else if (page.includes("Your reports")) 
+        botData.details = "Watching made reports";
+       else if (page.includes("Queue")) 
+        botData.details = "[Staff] Viewing the queue";
+       else if (page.includes("Pending Reports")) 
+        botData.details = "[Staff] Watching pending reports";
+       else 
+        botData.details = "Editing profile";
+      
+
```
Comment (ImRodry) [inline websites/D/Discord BotList/presence.ts:118]: ditto

**Review by ImRodry — COMMENTED** [review]
In general just format the entire file with prettier and you should be good to go

**Pull request closed without merging**